R/utilities.R

```diff
@@ -0,0 +1,346 @@
+library(tidyverse)
+library(behindbarstools)
+library(docstring)
+
+read_new_fac_info <- function(google_sheet_url = NULL) {
+    #' Read in UCLA facility data updates 
+    #'
+    #' Reads in new entries for the facility info sheet from Google Sheets.  
+    #'
+    #' @param google_sheet_url character string URL for the Google Sheet 
+    #'
+    #' @return data frame 
+    
+    if (is.null(google_sheet_url)) {
+        google_sheet_url <- "https://docs.google.com/spreadsheets/d/1tAhD-brnuqw0s55QXM-xYLPsyD-rNrqHbAVIbxSOMwI/edit#gid=1597803218"
+    }
+    
+    google_sheet_url %>% 
+        googlesheets4::read_sheet(sheet = "fac_data") %>% 
```

Comment:
Noting that this package is experimental; looks like this package corresponds to Google Sheets API version 4. If  the API version upgrades to 5, I wonder if we'll be able to continue using v4.

Reply:
Yes! I totally agree - hmm two initial thoughts (1) we can use [renv](https://rstudio.github.io/renv/) or something similar to manage our package versions, or (2) we can assume that if/when there's a v5, it'll have a `read_sheet` function (a Google sheets integration package that can't read Google sheets seems... not ideal). Very open to any other suggestions though!!

Reply:
Oooh I like the idea of using renv! I have not done that but it looks cool

Reply:
API version 3 is still supported by google at least until June 2021. We should have substantial time between the migration to API version 5 and when 4 gets shut down so I think its not an issue for this project. As long as we have some version documentation system for the package requirements I think thats good.